### PR TITLE
modify workbench create org rule and some jump link

### PR DESCRIPTION
**Pull request by chengjoey**
#### What type of this PR

Add one of the following kinds:
/kind bug

#### What this PR does / why we need it:
fix workbench link bug,create org pattern

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda-org.erda.cloud/erda/workBench/projects/387/issues/all?id=71540&issueFilter__urlQuery=eyJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciLCJXT05URklYIiwiUkVPUEVOIl0sImFzc2lnbmVlSURzIjpbIjEwMDEyMDUiXX0%3D&issueTable__urlQuery=eyJwYWdlTm8iOjF9&issueViewGroup__urlQuery=eyJ2YWx1ZSI6ImthbmJhbiIsImNoaWxkcmVuVmFsdWUiOnsia2FuYmFuIjoiZGVhZGxpbmUifX0%3D&iterationID=429&type=BUG)

### CODE REVIEW

File: modules/openapi/component-protocol/scenarios/home-page-content/components/emptyOrgContent/render.go [hunk 1]
```diff
@@ -114,7 +114,7 @@ func (e *EmptyOrgContent) Render(ctx context.Context, c *apistructs.Component, s
 	}, Value{
 		Props: PropValue{
 			RenderType: "linkText",
-			Title:      i18nLocale.Get(i18n.I18nKeyOrgCreate),
+			Title:      fmt.Sprintf("* %s", i18nLocale.Get(i18n.I18nKeyOrgCreate)),
```
Comment (Effet) [inline modules/openapi/component-protocol/scenarios/home-page-content/components/emptyOrgContent/render.go:117]: paste an screenshot in comment, let me know how it looks like

File: modules/openapi/component-protocol/scenarios/home-page-sidebar/components/createProjectTip/render.go [hunk 2]
```diff
@@ -137,7 +137,7 @@ func (p *CreateProjectTip) Render(ctx context.Context, c *apistructs.Component,
 		"createProjectDoc": {
 			Command: Command{
 				Key:     "goto",
-				Target:  "https://docs.erda.cloud/1.0/manual/platform-design.html#%E7%A7%9F%E6%88%B7-%E4%BC%81%E4%B8%9A",
+				Target:  "https://docs.erda.cloud/1.0/manual/platform-design.html#%E9%A1%B9%E7%9B%AE%E5%92%8C%E5%BA%94%E7%94%A8",
```
Comment (Effet) [inline modules/openapi/component-protocol/scenarios/home-page-sidebar/components/createProjectTip/render.go:140]: version of docs should be configrable

File: modules/openapi/component-protocol/scenarios/home-page-sidebar/components/emptyOrgText/render.go [hunk 3]
```diff
@@ -113,7 +113,7 @@ func (this *EmptyOrgText) Render(ctx context.Context, c *apistructs.Component, s
 	this.Operations["toJoinOrgDoc"] = Operation{
 		Command: Command{
 			Key:     "goto",
-			Target:  "https://docs.erda.cloud/1.0/manual/platform-design.html#%E7%A7%9F%E6%88%B7-%E4%BC%81%E4%B8%9A",
+			Target:  "https://docs.erda.cloud/1.0/manual/platform-design.html#%E7%A7%9F%E6%88%B7-%E7%BB%84%E7%BB%87",
```
Comment (Effet) [inline modules/openapi/component-protocol/scenarios/home-page-sidebar/components/emptyOrgText/render.go:116]: version of docs should be configrable

File: modules/openapi/component-protocol/scenarios/home-page-sidebar/components/orgFormModal/render.go [hunk 4]
```diff
@@ -101,8 +101,8 @@ func (o *OrgFormModal) SetProps() {
 				"required":  true,
 				"rules": []interface{}{
 					map[string]interface{}{
-						"msg":     "由小写字母、数字，-组成",
-						"pattern": "/^[a-z0-9-]*$/",
+						"msg":     "由小写字母、数字，-组成(不允许纯数字)",
+						"pattern": "/^(?![0-9]+$)[a-z0-9-]*$/",
```
Comment (Effet) [inline modules/openapi/component-protocol/scenarios/home-page-sidebar/components/orgFormModal/render.go:105]: is empty string allowed?

**Review by Effet — CHANGES_REQUESTED** [review]

File: modules/openapi/component-protocol/scenarios/home-page-content/components/emptyOrgContent/render.go (hunk 1 above)
Comment (chengjoey) [reply modules/openapi/component-protocol/scenarios/home-page-content/components/emptyOrgContent/render.go:117]: image

File: modules/openapi/component-protocol/scenarios/home-page-sidebar/components/orgFormModal/render.go (hunk 4 above)
Comment (chengjoey) [reply modules/openapi/component-protocol/scenarios/home-page-sidebar/components/orgFormModal/render.go:105]: not allowed
image
and backend will check org name

File: modules/openapi/conf/conf.go [hunk 5]
```diff
@@ -60,6 +60,9 @@ type Conf struct {
 
 	// Allow people who are not admin to create org
 	CreateOrgEnabled bool `default:"false" env:"CREATE_ORG_ENABLED"`
+
+	// DocVersion version of document
+	DocVersion string `default:"1.0" env:"DOC_VERSION"`
```
Comment (Effet) [inline modules/openapi/conf/conf.go:65]: I think we already have some way to get base version of erda, no need to add a new env.

**Review by Effet — CHANGES_REQUESTED** [review]

File: pkg/strutil/regexp.go [hunk 6]
```diff
@@ -42,6 +42,13 @@ func IsValidPrjOrAppName(repo string) bool {
 	return reg.MatchString(repo)
 }
 
+var orgReg = regexp.MustCompile(`\d*[a-z-][a-z\d-]*`)
+
+// IsValidOrgName check org name can contain a-z0-9- but can not pure 0-9
+func IsValidOrgName(repo string) bool {
+	return orgReg.MatchString(repo)
+}
+
```
Comment (Effet) [inline pkg/strutil/regexp.go:51]: should move to core-services, strutil only for common case

**Review by Effet — CHANGES_REQUESTED** [review]

File: modules/core-services/utils/regexp.go [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+// FITNESS FOR A PARTICULAR PURPOSE.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+package utils
+
+import "regexp"
+
+var orgReg = regexp.MustCompile(`\d*[a-z-][a-z\d-]*`)
```
Comment (Effet) [inline modules/core-services/utils/regexp.go:18]: please check "-123" or "123-" should not valid

**Review by Effet — CHANGES_REQUESTED** [review]

**Review by Effet — APPROVED** [review]

**Pull request merged**